docs/sources/medgen.md

```diff
@@ -0,0 +1,19 @@
+# MONDO - MedGen Alignment
+
+**Source name:** MedGen
+
+**Source description:** Organizes information related to human medical genetics, such as attributes of conditions with a genetic contribution. 
+
+**Homepage:** https://www.ncbi.nlm.nih.gov/medgen/
+
+## Comments about this source:
+### Mappings
+Mappings are created from [MedGenIDMappings.txt.gz](https://ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt.gz) downloadable from MedGen's [FTP site](https://ftp.ncbi.nlm.nih.gov/pub/medgen/).
+
+Predicates: All mappings in that file can be interpreted as `skos:exactMatch`, per verbal confirmation with MedGen staff.
```

Comment:
## MedGen docs
I added docs for the MedGen source. Let me know if you want anything rephrased, etc.